samples/hello-connector-solace/src/main/java/com/solace/quarkus/samples/HelloConsumer.java

```diff
@@ -35,13 +40,34 @@ Multi<Message<String>> publishMessage() {
      * @param p
      */
     @Incoming("hello-in")
-    @Acknowledgment(Acknowledgment.Strategy.MANUAL)
-    CompletionStage<Void> consumeMessage(SolaceInboundMessage<?> p) {
-        Log.infof("Received message: %s from topic: %s", new String(p.getMessage().getPayloadAsBytes(), StandardCharsets.UTF_8),
+    CompletionStage<Void> consumeInboundMessage(SolaceInboundMessage<String> p) {
```

Comment:
👍 for removing acknowledgement annotation. Just noting that ll get into conflict with #48